library/src/G4CMPPhononElectrode.cc

```diff
@@ -145,7 +146,8 @@ ProcessAbsorption(const G4Track& track, const G4Step& step, G4double EDep,
     do {
       reflectedKDir = G4CMP::LambertReflection(surfNorm);
     } while (!G4CMP::PhononVelocityIsInward(theLattice, pol,
-                                            kmag*reflectedKDir, surfNorm));
+                                            kmag*reflectedKDir, surfNorm,
+                                            track.GetPosition()));
```

Comment:
See my comment about moving this to G4CMPUtils rather than duplicating the code from BoundaryProcess.